ast/builtins.go

```diff
@@ -959,6 +961,44 @@ The set of regex symbols is limited for this builtin: only ` + "`.`, `*`, `+`, `
  */
 var stringsCat = category("strings")
 
+var AnyPrefixMatchesAnyString = &Builtin{
+	Name:        "any_prefix_matches_any_string",
+	Description: "Returns a boolean indicating whether any of the prefixes matches any of the strings.",
+	Decl: types.NewFunction(
+		types.Args(
+			types.Named("strings", types.NewAny(
+				types.NewSet(types.S),
+				types.NewArray(nil, types.S),
+			)).Description("strings to match against"),
+			types.Named("prefixes", types.NewAny(
+				types.NewSet(types.S),
+				types.NewArray(nil, types.S),
+			)).Description("prefixes"),
+		),
+		types.Named("result", types.B).Description("result of the prefix check"),
```

Comment:
Instead of "result of the prefix check" I'd prefer something like `true if <condition for this to be true>, otherwise false`

Reply:
fixed